decidim-admin/app/helpers/decidim/admin/officializations_helper.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Admin
+    module OfficializationsHelper
+      def profile_path(user)
+        decidim.profile_path(user&.nickname)
```

Comment:
We can switch to implicit style here, too. And at that point, I don't think the helper adds much, so we can just inline the call in the views?

Reply:
You're right, I'm refactoring.

Reply:
That's interesting, I think it's because not redefining `to_param` in the model. Maybe we should do it, but definitely not in this PR :)